templates/swift/Sources/WebSockets/WebSocketClient.swift.twig

```diff
@@ -246,7 +246,9 @@ public class WebSocketClient {
         
         let basicUpgrader = NIOWebSocketClientUpgrader(
             requestKey: self.frameKey,
-            upgradePipelineHandler: self.upgradePipelineHandler
+            upgradePipelineHandler: { channel, response in
+                self.upgradePipelineHandler(channel: channel, response: response)
+            }
```

Comment:
What does this change do?

Reply:
<img width="968" alt="Bildschirmfoto 2024-02-24 um 08 13 32" src="https://github.com/appwrite/sdk-generator/assets/46761404/64a2bd97-c453-4d0a-bc6e-d57295c74493">
<img width="968" alt="Bildschirmfoto 2024-02-24 um 08 13 43" src="https://github.com/appwrite/sdk-generator/assets/46761404/96de2614-af3b-43f8-b715-7e31cf0c237a">

As seen in the images above, it produces a warning if you just pass the function as an argument. This warning can be silenced if you wrap the function in a closure. I am using Version 15.3 beta 3 (15E5194e) btw.

Reply:
I see, let's add the `@Sendable` attribute to the `upgradePipelineHander` function instead. That will do the same thing but be more explicit

Reply:
<img width="952" alt="Bildschirmfoto 2024-02-25 um 11 06 29" src="https://github.com/appwrite/sdk-generator/assets/46761404/39a9feec-c9af-4f08-8b60-28413ef1df22">

This produces yet another warning.

Reply:
<img width="1020" alt="Bildschirmfoto 2024-02-25 um 11 08 20" src="https://github.com/appwrite/sdk-generator/assets/46761404/efd6ec12-8e73-4405-8499-683d543b28b4">

And when I mark the class as sendable, I get this warning.

Reply:
Interesting, in that case let's go with your original approach :+1: